package.json

```diff
@@ -1,13 +1,25 @@
 {
   "name": "tangram-cartocss",
-  "version": "0.6.0",
+  "version": "1.0.0",
```

Comment:
I would keep version <1.0.0 as we might want to change the interface.

Reply:
Yes, "1.0.0" was the default value, do you think that "0.7.0" is OK?

Reply:
Yes. 😈